packages/wallets/src/features/accounts/modules/Poa/components/POAMobile/POAMobile.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useMemo, useState } from 'react';
+import classnames from 'classnames';
+import { useFormikContext } from 'formik';
+import { LabelPairedArrowLeftMdBoldIcon } from '@deriv/quill-icons';
+import { Localize, useTranslations } from '@deriv-com/translations';
+import { InlineMessage, Text } from '@deriv-com/ui';
+import { ModalStepWrapper } from '../../../../../../components';
+import { Footer } from '../../../components';
+import { usePoa } from '../../hooks';
+import { TPoaValues } from '../../types';
+import { AddressSection } from '../AddressSection';
+import { DocumentSubmission } from '../DocumentSubmission';
+import './POAMobile.scss';
+
+const ProgressBar = ({ isActive }: { isActive?: boolean }) => (
+    <div className='wallets-progress-container'>
+        <div className={classnames('wallets-progress-bar', { 'wallets-progress-bar--animate': isActive })} />
+    </div>
+);
+
+type POAMobileProps = {
+    countryCode: string;
+    onCompletion?: VoidFunction;
+};
+
+const POAMobile: React.FC<POAMobileProps> = ({ countryCode, onCompletion }) => {
+    const { localize } = useTranslations();
+    const { errorSettings } = usePoa();
+    const { errors, isValid, values } = useFormikContext<TPoaValues>();
+    const [shouldRenderDocumentUpload, setShouldRenderDocumentUpload] = useState(false);
+
+    const isNextBtnDisabled = useMemo(() => {
+        if (shouldRenderDocumentUpload) {
+            return false;
+        }
+        return (
+            !values.firstLine ||
+            !!errors.firstLine ||
+            !!errors.secondLine ||
+            !values.townCityLine ||
+            !!errors.townCityLine ||
+            !!errors.stateProvinceLine ||
+            !!errors.zipCodeLine
+        );
+    }, [values, errors, shouldRenderDocumentUpload]);
+
+    if (shouldRenderDocumentUpload) {
+        return (
+            <ModalStepWrapper
+                disableAnimation
+                renderFooter={() => (
+                    <Footer disableNext={!isValid} nextText={localize('Submit')} onClickNext={onCompletion} />
+                )}
+                title={localize('Add a real MT5 account')}
+            >
+                <div className='wallets-poa-mobile-layout'>
+                    <div className='wallets-poa-mobile-layout__header'>
+                        <div className='wallets-timeline'>
+                            <LabelPairedArrowLeftMdBoldIcon onClick={() => setShouldRenderDocumentUpload(false)} />
+                            <Text align='start' as='p' lineHeight='sm' size='sm'>
+                                <Localize
+                                    components={[<strong key={0} />]}
+                                    i18n_default_text='<0>Step 2/2:&nbsp;</0> Upload proof of address'
```

Comment:
&nbsp?

Reply:
To add a space character between localize text and normal text